### PR TITLE
fix: missing mulitiple metrics on pivot operator

### LINKED ISSUE #16023: [time-series]fail to add more than one metric in time-series chart
Suspecting this is a regression from a recent fix #15975 

https://user-images.githubusercontent.com/67837651/127873306-d13ff760-03c7-4d9f-8b7d-e341697494fd.mov

### PR DESCRIPTION
**Pull request by zhaoyongjie**
### SUMMARY

fixes: https://github.com/apache/superset/issues/16023
closes: https://github.com/apache/superset/issues/16023

### After
image

### TESTING INSTRUCTIONS

UT converage

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/utils/pandas_postprocessing.py [hunk 1]
```diff
@@ -271,8 +271,12 @@ def pivot(  # pylint: disable=too-many-arguments
     series_set = set()
     if not drop_missing_columns and columns:
         for row in df[columns].itertuples():
-            metrics_and_series = tuple(aggfunc.keys()) + tuple(row[1:])
-            series_set.add(str(metrics_and_series))
+            metrics_and_series = []
+            for metric in aggfunc.keys():
+                metrics_and_series.append(tuple([metric]) + tuple(row[1:]))
+            for m_series in metrics_and_series:
```
[the remark this answers is not in the document]
Comment (villebro) [reply superset/utils/pandas_postprocessing.py:277]: Can we just do it in one pass, like 
```python
            for metric in aggfunc.keys():
                series_set.add(str(tuple([metric]) + tuple(row[1:])))
```

**Review by villebro — APPROVED** [review]
Thanks for the quick fix - minor non-blocking stylistic proposal

File: superset/utils/pandas_postprocessing.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (zhaoyongjie) [reply superset/utils/pandas_postprocessing.py:277]: nice! thanks!

**Review by suddjian — APPROVED** [review]
lgtm

**Comment by jinghua-qa** [comment]
I have tried in the test env with dd on,  I can add multi metrics but DATA panel is not scrollable, and user need to click collapse/expand button in order to see the rest of the filed in DATA and for south tables

**Comment by junlincc** [comment]
lgtm 
https://user-images.githubusercontent.com/67837651/127907559-38b6326d-247c-4082-87ab-f88de0e08490.mov

**Pull request merged**